src/scss/components/_dropdown.scss

```diff
@@ -12,12 +12,15 @@ $icon-world: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org
 $animation-toggle: transform 0.2s ease-in-out;
```

Comment:
En la linea **uno** el comentario deberia ser: /* === Custom variables and mixins === */ no ?
Tambien quiza dejaria las variables y despues el mixin ? jaja